include/openssl/hpke.h

```diff
@@ -1,7 +1,8 @@
 /*
- * Copyright 2022-2023 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2000-2023 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright Siemens AG 2018-2020
  *
- * Licensed under the OpenSSL license (the "License").  You may not use
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
```

Comment:
NOTE: this file was introduced in 2022, saying "the OpenSSL license". Could we add something to check that (new) files have the Apache 2.0 licence? (not in this PR - asking generally)

Reply:
Great Point @tom-cosgrove-arm 

Sure, I have opened a new bug - https://github.com/openssl/openssl/issues/23604 to address the same. Thanks!